.travis.yml

```diff
@@ -23,6 +23,7 @@ matrix:
       - unset CC ; unset CXX ; unset CC_FOR_BUILD ; unset CXX_FOR_BUILD
       # homebrew addon is just broken
       - HOMEBREW_NO_AUTO_UPDATE=1 brew install yasm
+      - HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall json-c
```

Comment:
Why is this needed?

Reply:
fontconfig build may fail in macOS with that json-c version, and it seems no configure options to disable it, just uninstall it that can pass as well

Reply:
In general we write patches to disable such stuff, because if we need a library dependency, it needs to be statically linked into the resulting engine for this engine to work on consumer systems.

But in any case, I believe fontconfig is unnecessary. Please see my answer below.